py3status/modules/static_string.py

```diff
@@ -3,35 +3,31 @@
 Display static text.
 
 Configuration parameters:
-    format: text that should be printed (default '')
+    format: default format for this module (default 'hello, world!')
```

Comment:
We can do `format = '\?color=#ff0&show Hello, world!'` here... so we can use `default format for this module` here even if it does not contain any fancy formatting tricks... or is that still wrong?  Anyhow, I updated the string to `Hello, world!`.